app/main.py

```
@@ -353,13 +354,50 @@ def update_jdk_metrics():
+                last_build = get_last_build_status(cfg["id"])
+                status = last_build['status']
+                status_value = {"SUCCESS": 1, "FAILURE": 0, "NO_BUILDS": -1}.get(status, -1)
+
+                BUILD_STATUS_GAUGE.labels(
+                    template_id=template_id,
+                    build_type_name=cfg["name"],
+                    build_type_id=cfg["id"],
+                    build_url=cfg["webUrl"]
+                ).set(status_value)
+
+        logging.info(f"Build status metrics updated successfully")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)